nvflare/fuel/hci/client/api_spec.py

```diff
@@ -84,6 +84,10 @@ def get_meta(self):
         return self.get_prop(CommandCtxKey.META)
 
 
+class ApiPocValue(object):
+    ADMIN = "admin"
```

Comment:
I think it is the naming that is causing the confusion.

We should just remove all "POC" word in API implementation.